packages/editor/src/components/post-actions/actions.js

```diff
@@ -1026,6 +1258,8 @@ export function usePostActions( postType, onActionPerformed ) {
 				: false,
 			! isTemplateOrTemplatePart && renamePostAction,
 			isTemplateOrTemplatePart && renameTemplateAction,
+			isPattern && exportPatternAsJSONAction,
+			isPattern && deletePatternAction,
```

Comment:
I'm assuming these two actions were just copied as is right?

Reply:
Why did you choose to merge the "reset" action but not the "delete" one?

Reply:
> I'm assuming these two actions were just copied as is right?
Yes mostly copied as is with some very minor changes like stores used imports etc...

> Why did you choose to merge the "reset" action but not the "delete" one?

The reset template was mostly ready to include template parts it even included error messages related to the template parts. The delete pattern is more complex it may make the reusable blocks store. I think it is better to have its own PR.